changelog/669.feature.rst

```diff
@@ -0,0 +1 @@
+|commands| Change :func:`commands.register_injection <ext.commands.register_injection>` to now return an instance of :class:`Injection <ext.commands.Injection>`.
```

Comment:
should this be labeled as a breaking change?

Reply:
I think returning something when we used to return None not a breaking change.

Reply:
https://github.com/DisnakeDev/disnake/pull/675#discussion_r950658142

Reply:
:^)

Reply:
https://github.com/DisnakeDev/disnake/pull/675/commits/1c35cd0d94975673acbb913aa8639206235e11e8